docs/DeveloperGuide.md

```diff
@@ -1479,80 +1479,113 @@ otherwise)
 **MSS:**
 
 1. Logged-in Owner types command to view a list of deliveries.
-2. DMS displays a list of all deliveries sorted in descending expected delivery date (newest to oldest).
+2. DMS displays a list of all deliveries sorted in descending expected delivery date (latest to earliest).
 
    Use Case Ends.
 
 **Extensions:**
 
-* 1a. User specifies status field in command.
-    * 1a1. DMS display a list of deliveries filtered by the specified status.
+- 1a. User specifies status field in command.
+
+    - 1a1. DMS display a list of deliveries filtered with the specified status.
 
       Use Case Ends.
 
-* 1b. User specifies customer field in command.
-    * 1b1. DMS displays a list of deliveries filtered by the specified customer.
+- 1b. User specifies customer field in command.
+
+    - 1b1. DMS displays a list of deliveries filtered by the specified customer id.
 
       Use Case Ends.
 
-* 1c. User specifies expected delivery date field in command.
-    * 1c1. DMS displays a list of deliveries filtered by the specified expected delivery date.
+- 1c. User specifies expected delivery date field in command.
+
+    - 1c1. DMS displays a list of deliveries filtered by the specified expected delivery date.
 
       Use Case Ends.
 
-* 1d. User specifies both status and customer fields.
-    * 1c1. DMS displays a list of deliveries filtered by the specified status and customer.
+- 1d. User specifies expected delivery date field as "TODAY" in command.
+
+    - 1d1. DMS displays a list of deliveries filtered by the expected delivery date that is today's date.
```

Comment:
Not sure if this would be considered giving too much details.
Similarly for 1g, 1j, 1k

Reply:
Like "TODAY" is it?

Reply:
Yep

Reply:
Hmm but because our `--date` accepts either `TODAY` or an actual date, it could be quite different?